modules/spring-config/src/main/java/org/mule/runtime/config/spring/MuleArtifactContext.java

```diff
@@ -52,50 +73,44 @@
  */
 public class MuleArtifactContext extends AbstractXmlApplicationContext
 {
+
+    /**
+     * Indicates that XSD validation should be used (found no "DOCTYPE" declaration).
+     */
+    private static final int VALIDATION_XSD = 3;
```

Comment:
Yes, but I don't want to tied this to spring. I tried to use one from the XSD api.